vtex/faststore
# fix: Consider `shouldSplitItem` value from `ValidateCart` mutation response

## What's the purpose of this pull request?

This PR intends to fix the behavior of `shouldSplitItem` value sent by through the `ValidateCart` mutation.

Currently, the variable value is being overridden after some mutation calls due to its value not being sent back in the mutation response, so with these changes we will consider the same value set in the store config.

## How to test it?

Perform actions that triggers the `ValidateCart` mutation (open/close the cart, add/remove items to/from the cart) and check if the `shouldSplitItem` variable is being sent on the payload and if its value is corresponding to the one set in the store config.

### Starters Deploy Preview



## References

Issue: #2426
